analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Invocation.cs

```diff
@@ -884,11 +884,11 @@ public void Invocation_TargetMethodIsDelegateInvoke()
         }
 
         [DataTestMethod]
-        [DataRow("null", "null", true, "Null", "Null")]
-        [DataRow("null", "new object()", false, "Null", "NotNull")]
-        [DataRow("new object()", "null", false, "NotNull", "Null")]
-        [DataRow("new int?()", "null", false, "NotNull", "Null")]   // Should be "true" and "Null" because new int()? is still null
-        [DataRow("new int?(42)", "null", false, "NotNull", "Null")]
+        [DataRow("null", "null", true, "ObjectNull", "ObjectNull")]
```

Comment:
This is annoying :( Let's override it in `ObjectConstraint` and `BoolConstraint` to return only `Null`, `NotNull`, `True`, `False` as before.

Others are fine (like `LockHeld`).